Makefile

```diff
@@ -23,29 +37,33 @@ restart-server:
 build: node_modules
 	$(NPM) run build
 
-## dist: creates the bundle file for aws deployment
-dist-aws: build
-	rm -rf aws/$(app_id) && mkdir -p aws/$(app_id)
-	mv dist/* aws/$(app_id)
-	mv node_modules aws/$(app_id)
-	cp -r src/locales aws/$(app_id)
-	rm -r dist
-	cp src/manifest.json aws
-	cp -r static aws
-	cd aws ; \
-		zip -rm $(app_id).zip $(app_id) ; \
-		zip -rm ../$(app_id).zip manifest.json static $(app_id).zip
-	rm -r aws
+## dist: creates the bundle file for dev deployment
+dist-dev: build
+	rm -rf $(DIST_DIR)/$(BUNDLE_NAME) && mkdir -p $(DIST_DIR)/$(BUNDLE_NAME)
+	mv $(TS_DIST_DIR)/* $(DIST_DIR)/$(BUNDLE_NAME)
+	rm -r $(TS_DIST_DIR)
+	mv node_modules $(DIST_DIR)/$(BUNDLE_NAME)
+	cp -r src/locales $(DIST_DIR)/$(BUNDLE_NAME)
+	cp src/manifest.json $(DIST_DIR)
+	cp -r static $(DIST_DIR)
+	cd $(DIST_DIR) ; \
+		zip -rm $(BUNDLE_NAME).zip $(BUNDLE_NAME) ; \
```

Comment:
Does the `zip` file inside the bundle need to match the `name` from `aws_lambda.functions` in the manifest? 
![Screenshot from 2022-11-30 12-52-54](https://user-images.githubusercontent.com/16541325/204789727-eb497524-be2f-412a-82a8-6d5ae3f6ff57.png)

Reply:
Yes, it should be

Reply:
So this will not work to deploy it automatically, only to share the dist with the correct versioning

Reply:
Maybe I'm missing some here. The bundle created via `make dist-dev` is not deployable via `appsctl` because the bundle name doesn't match.